nibabel/cifti2/parse_cifti2.py

```diff
@@ -121,33 +121,9 @@ def _chk_pixdims(hdr, fix=False):
 
 
 class _Cifti2AsNiftiImage(Nifti2Image):
+    """ Load a NIfTI2 image with a Cifti2 header """
     header_class = _Cifti2AsNiftiHeader
-    files_types = (('image', '.nii'),)
-    valid_exts = ('.nii',)
     makeable = False
-    rw = True
-
-    def __init__(self, dataobj, affine, header=None,
```

Comment:
The point of the subclass is to be a `Nifti2Image` with [this header](https://github.com/effigies/nibabel/blob/ad353403ef2747e9813c4ea4c8a666742bf0817e/nibabel/cifti2/parse_cifti2.py#L79-L120), which has different constraints from a standard `Nifti2Header`.